demofile-parsing/Round.js

```diff
@@ -115,37 +115,35 @@ class Round {
 
   getKills() {
     // Keeping a ledger of the name of the killer for each kill in the round.
-    var roundKillerNames = [];
-    for (var i = 0; i < this.keyEvents.length; i++) {
+    // var roundKillerNames = [];
+    // for (var i = 0; i < this.keyEvents.length; i++) {
 
-      // If the event is a player death add the killer to the array.
-      if (this.keyEvents[i].type == 'player_death') {
-        roundKillerNames.push(this.keyEvents[i].attacker_name);
-      }
-    }
-    this.countKills(roundKillerNames);
+    //   // If the event is a player death add the killer to the array.
+    //   if (this.keyEvents[i].type == 'player_death') {
+    //     roundKillerNames.push(this.keyEvents[i].attacker_name);
+    //   }
+    // }
+    // this.countKills(roundKillerNames);
 
     // TODO (Figure out why the below chunk is causing async nightmares).
     // To test the below comment out the above part of getKills() and uncomment the below.
 
-    // for (var k = 0; k < this.keyEvents.length; k++) {
+    for (var k = 0; k < this.keyEvents.length; k++) {
 
     //   // If the event is a player death add the killer to the array.
-    //   if (this.keyEvents[k].type == 'player_death') {
-    //     killerName = this.keyEvents[k].attacker_name;
```

Comment:
You need to add `var killerName`. Easy to miss this especially coming from python. Also for some reason there are no errors thrown 🤔 Not sure why but it would have made this a lot easier to debug.